recipes/avahi/avahi_0.6.31.oe

```diff
@@ -1,8 +0,0 @@
-require avahi.inc
-SRC_URI += "\
-file://reuseport-check.patch \
-file://0001-avahi-fix-avahi-status-command-error-prompt.patch \
-file://out-of-tree.patch \
-file://avahi_fix_install_issue.patch \
-file://fix_for_automake_1.12.x.patch \
-"
```

Comment:
out-of-tree.patch: fixed in 0.6.32
avahi_fix_install_issue.patch: fixed in 0.6.32
fix_for_automake_1.12.x.patch: fixed in 0.6.32

removing them from files/ dir.